src/cache/packages.rs

```diff
@@ -70,8 +70,7 @@ impl PackagesCache {
 impl Default for PackagesCache {
     fn default() -> Self {
         let directory = {
-            let mut home = env::var("HOME").unwrap_or_else(|_| ".".to_string());
-            home.push_str(PACKAGES_CACHE_FOLDER);
+            let home = my_home().unwrap().unwrap().join(PACKAGES_CACHE_FOLDER);
```

Comment:
This is Windows OS Home path right ?
And this could break execution on Unix systems. Can we have a separate functions with macros that'll return home path?

Something like this

```rs
#[cfg(target_os = "windows")]
fn get_home() -> String {
    std::env::var("USERPROFILE").unwrap_or_else(|_| "C:\\Users\\Default".to_string())
}

#[cfg(not(target_os = "windows"))]
fn get_home() -> String {
    std::env::var("HOME").unwrap_or_else(|_| "/home/default".to_string())
}
```

Reply:
Reference: https://github.com/ljtpetersen/homedir/blob/master/src/lib.rs#L118-L136
It seems `homedir::my_home` is doing all the tricks for us

Reply:
Yes that is the cool part about it. I cleaned up the code in another pr.